### PR TITLE
add forcepoint integraion

### PR DESCRIPTION
**Pull request by noaco**
fixes [etc/issue#6067](https://github.com/demisto/etc/issues/6067)

### CODE REVIEW

**Comment by noaco** [comment]
@mayairv [Forcepoint documentation.docx](https://github.com/demisto/content/files/1595410/Forcepoint.documentation.docx)

**Comment by noaco** [comment]
@mayairv updated the documentation 
[Forcepoint documentation.docx](https://github.com/demisto/content/files/1597793/Forcepoint.documentation.docx)

File: Integrations/integration-Forcepoint.yml [hunk 1]
```diff
@@ -0,0 +1,573 @@
...
+description: Advanced threat protection with added local management controls.
+configuration:
+- display: Server URL (e.g. https://192.168.0.1)
+  name: url
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: User name
```
Comment (orius123) [inline Integrations/integration-Forcepoint.yml]: Just a minor comment, if it holds both username and password display should be credentials and not User name

**Comment by anara123** [comment]
@noaco For documentation we open new issue. So please open new issue with `ForcePoint Documentation` title, and relate this issue link there. assign @mayairv  and upload the docs there

**Pull request merged**